papaparse.js

```diff
@@ -1516,7 +1517,11 @@ License: MIT
 				for (var i = 0; i < rows.length; i++)
 				{
 					row = rows[i];
-					cursor += row.length;
+					if (config.header && i === 0 && firstLine !== undefined) {
```

Comment:
Can you add a comment like: 
```
// use firstline as row length may be changed due to duplicated headers
```
before the if clause

Reply:
Thanks. I fixed that.